lms/djangoapps/courseware/views/views.py

```diff
@@ -1648,3 +1611,20 @@ def get_financial_aid_courses(user):
             )
 
     return financial_aid_courses
+
+
+def redirect_if_needed_to_access_course(request, course):
```

Comment:
Nit: This is named kind of confusingly. It makes it seem like you're redirecting the user to access the course or something like that?